pkg/system/sysinfo.go

```diff
@@ -3,9 +3,50 @@
 package system
 
 import (
+	"io/ioutil"
+	"os"
 	"syscall"
+
+	"golang.org/x/sys/unix"
 )
 
+// SysInfo defines system info on current machine
+type SysInfo struct {
```

Comment:
In the built-in package, Sysinfo refers to http://docs.studygolang.com/pkg/syscall/#Sysinfo_t : 
```
type Sysinfo_t struct {
    Uptime    int64
    Loads     [3]uint64
    Totalram  uint64
    Freeram   uint64
    Sharedram uint64
    Bufferram uint64
    Totalswap uint64
    Freeswap  uint64
    Procs     uint16
    Pad       uint16
    Pad_cgo_0 [4]byte
    Totalhigh uint64
    Freehigh  uint64
    Unit      uint32
    X_f       [0]byte
    Pad_cgo_1 [4]byte
}
```

So, I am wondering if it is proper to change the definition of `SysInfo`. @Ace-Tang

Reply:
This is not proper, also I recognize the file name is also not proper, it should be `system.go`, and change the `Sysinfo` to `Info`. That will make sense. @allencloud , WDYT.

Reply:
Discussed offline. And @Ace-Tang would do some update.